### ISSUE TITLE
Bibliographic coupling for authors

### ISSUE THREAD

**Issue by sjgknight** [massimoaria/bibliometrix#269]
Biblioshiny has a very nice interface for identifying common references using bibliographic coupling (thank you!).  I think often specific references are too granular, and instead we are interested in coupling by author (i.e., you and I both cite J, Doe - even if they're different works).  I'm trying to:
1. find if there is a way using the functions to extract these common authors, in the same way as the biblioshiny interface provides the table?
2. somewhat separately, easy ways to exclude the authors (e.g., me and you) from the analysis (I know I can use remove.terms for this)

Is '1' hidden somewhere in a matrix I might extract myself?

**Comment by massimoaria** [comment]
> Biblioshiny has a very nice interface for identifying common references using bibliographic coupling (thank you!). I think often specific references are too granular, and instead we are interested in coupling by author (i.e., you and I both cite J, Doe - even if they're different works). I'm trying to:
> 
> 1. find if there is a way using the functions to extract these common authors, in the same way as the biblioshiny interface provides the table?
Yes, it is possible to do that using the function biblioNetwork() in the R console.

> 2. somewhat separately, easy ways to exclude the authors (e.g., me and you) from the analysis (I know I can use remove.terms for this)

Not directly in Biblioshiny, but it is possible to remove authors from the network matrix (returned by biblioNetwork()), filtering its rows and columns.
 
> Is '1' hidden somewhere in a matrix I might extract myself?

**Comment by massimoaria** [comment]
Closing this as part of a review of the older issues.

bibliometrix has changed substantially since this was opened -- the importers, the network functions
and Biblioshiny have all been largely rewritten -- and from here there is no way to tell whether what
you described still happens.

If it does, please open a new issue with the version you are running and, where it applies, a file
that reproduces it. A fresh report with current details gets a faster and more useful answer than
reviving this thread.
